fedimint-rocksdb/src/lib.rs

```diff
@@ -234,6 +235,42 @@ impl<'a> IDatabaseTransactionOpsCore for RocksDbTransaction<'a> {
         }))
     }
 
+    async fn raw_find_by_range(&mut self, range: Range<&[u8]>) -> Result<PrefixStream<'_>> {
+        // turn an `iter` into a `Stream` where every `next` is ran inside
+        // `block_in_place` to offload the blocking calls
+        fn convert_to_async_stream<'i, I>(iter: I) -> impl futures::Stream<Item = I::Item>
```

Comment:
[nit] Good opportunity to move `convert_to_async_stream` higher then remove all the duplicate implementations